toolset/kover-cli/src/test/kotlin/kotlinx/kover/cli/md/MarkDownMain.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2000-2023 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.kover.cli.md
+
+import kotlinx.kover.cli.commands.RootCommand
+import java.io.PrintWriter
+
+fun main() {
```

Comment:
You can also drop a comment like 'Used for internal purposes to generate documentation, not an actual test'